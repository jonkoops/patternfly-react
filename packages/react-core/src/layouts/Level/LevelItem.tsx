import { HTMLProps, ReactNode, FunctionComponent } from 'react';

export interface LevelItemProps extends HTMLProps<HTMLDivElement> {
  /** content rendered inside the Level Layout Item */
  children?: ReactNode;
}

export const LevelItem: FunctionComponent<LevelItemProps> = ({ children = null, ...props }: LevelItemProps) => (
  <div {...props}>{children}</div>
);
LevelItem.displayName = 'LevelItem';
