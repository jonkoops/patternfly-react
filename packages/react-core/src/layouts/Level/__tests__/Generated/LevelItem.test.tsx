/**
 * @vitest-environment jsdom
 */
import { render } from '@testing-library/react';
import React from 'react';
import { expect, it } from 'vitest';

import { LevelItem } from '../../LevelItem';

it('LevelItem should match snapshot (auto-generated)', () => {
  const { asFragment } = render(<LevelItem children={<>ReactNode</>} />);
  expect(asFragment()).toMatchSnapshot();
});
