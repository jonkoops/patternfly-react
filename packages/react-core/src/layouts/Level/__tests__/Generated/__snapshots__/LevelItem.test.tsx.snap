// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LevelItem should match snapshot (auto-generated) 1`] = `
<DocumentFragment>
  <div>
    ReactNode
  </div>
</DocumentFragment>
`;
